terraform/create.go

```diff
@@ -163,6 +164,16 @@ func (t *Terraform) Create() error {
 }
 
 func (t *Terraform) preFlightCheck() error {
+	if os.Getenv("SSH_AUTH_SOCK") == "" {
+		return fmt.Errorf("ssh agent not running. Please run eval \"$(ssh-agent -s)\" and then ssh-add.")
+	}
+	if len(t.config.DeploymentConfiguration.DBPassword) <= 8 {
+		return fmt.Errorf("db password needs to be longer than 8 characters")
+	}
+	clusterName := t.config.DeploymentConfiguration.ClusterName
+	if len(clusterName) == 0 || clusterName[0] != '-' || !isAlphanumeric(clusterName) {
```

Comment:
I was almost certain you were going to use a regexp for this.

Reply:
lol .. haven't you heard of https://www.xkcd.com/1171/ ? :rofl: